core/src/stored/backends/crud_storage.cc

```diff
@@ -231,23 +234,31 @@ tl::expected<void, std::string> CrudStorage::test_connection()
 auto CrudStorage::stat(std::string_view obj_name, std::string_view obj_part)
     -> tl::expected<Stat, std::string>
 {
-  Dmsg1(debug_trace, "stat %s called\n", obj_name.data());
+  Dmsg1(debug_trace, "stat %s/%s called\n", obj_name.data(), obj_part.data());
```

Comment:
Not a fan of this tbh. Its too hard to misuse this function now.
```suggestion
  Dmsg1(debug_trace, "stat %.*s/%.*s called\n", saturating_cast<int>(obj_name.size()), obj_name.data(), saturating_cast<int>(obj_part.size()), obj_part.data());
```
where `saturating_cast<S>(U u)` returns
```
* S::max(), when u > (U)S::max()
* u, otherwise
```